DomainContent/Korea/Brush/Marker_Client.js

```diff
@@ -0,0 +1,557 @@
+                var oppositeHandMarkerID = this.getEquippedMarkerID(oppositeHand);
+
+                return oppositeHandMarkerID;
+            } else {
+                console.error("Error: Hand is neither right or left. Issue with utility method hasMarkerInOppositeHand.");
+            }
+        },
+        hasMarkerInHand: function (hand) {
+            // checks objects equipped to the hand
+            // if hand is equipped with a marker clone, will return the markerID
+            // else returns undefined
+
+            if (hand === LEFT || hand === RIGHT){
+                var curHand = hand === LEFT ? "LeftHand" : "RightHand";
+                var markerID = this.getEquippedMarkerID(curHand);
+        
+                return markerID;
+            } else {
+                console.error("Error: Hand is neither right or left. Issue with utility method hasMarkerInHand.");
+            }
+        }
+    };
+
+    var drawing = {
+        // handles the drawing methods
+        onStartLine: function (width){
+            if (DEBUG_MARKER_CLIENT){
+                console.info("startline");
+            }
+
+            var markerTipPos = utils.getMarkerTipPosition();
+
+            if (markerTipPos){
+                var position = JSON.stringify(markerTipPos);
+                var strWidth = String(width);    
+                var markerRotation = Entities.getEntityProperties(entityID, ["rotation"]).rotation;
+                var orientation = JSON.stringify(markerRotation);
+        
+                Entities.callEntityServerMethod(entityID, "startLine", [position, strWidth, orientation]);
+            } else {
+                console.error("Error: markerTipPosition has an issue in startLine.");
+            }
+        },
+
+        onDrawLine: function (width){
+    
+            var markerTipPos = utils.getMarkerTipPosition();
+
+            if (markerTipPos) {
+                var position = JSON.stringify(markerTipPos);
+                var strWidth = String(width);
+        
+                Entities.callEntityServerMethod(entityID, "drawLine", [position, strWidth]);
+            } else {
+                console.error("Error: markerTipPosition has an issue in drawLine.");
+            }
+        },
+
+        onFinishLine: function (width){
+
+            var markerTipPos = utils.getMarkerTipPosition();
+
+            if (markerTipPos) {
+                var position = JSON.stringify(markerTipPos);
+                var strWidth = String(width);
+        
+                Entities.callEntityServerMethod(entityID, "finishLine", [position, strWidth]);
+            } else {
+                console.error("Error: markerTipPosition has an issue in FinishLine.");
+            }
+        },
+
+        onEraseClosestLine: function (position){
+    
+            // Find entities with bounding box within search radius.
+            var entities = Entities.findEntities(position, ERASE_SEARCH_RADIUS);
```

Comment:
I would suggest using FindEntitiesByType here so you don't have to sort on the properties within this function